staging/src/k8s.io/kubectl/pkg/config/v1alpha1/types.go

```diff
@@ -24,13 +24,13 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 type Preference struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// overrides allows changing default flag values of commands.
+	// Overrides allows changing default flag values of commands.
```

Comment:
because we generate docs from these (https://github.com/kubernetes/website/blob/main/content/en/docs/reference/config-api/kuberc.v1alpha1.md), keep all of the godocs starting with the `json` attribute, lowercase, etc

Reply:
Update both v1alpha1 types and v1beta1 types such that the generated docs will properly use lowercase, but left the internal types with uppercase, since that's documenting go types.